src/OSPSuite.UI/Views/Importer/NanView.cs

```diff
@@ -57,5 +48,13 @@ public void AttachPresenter(INanPresenter presenter)
       {
          _presenter = presenter;
       }
+
+      public override void InitializeResources()
+      {
+         base.InitializeResources();
+         //TODO MVOE TO CONSTANTS
+         indicatorLayoutControlItem.Text = "NaN indicator".FormatForLabel();
```

Comment:
this is how label should be formated.  To be used every where
Only move the string to the constant file and use FormatForLabel() in the view